tox.ini

```diff
@@ -36,13 +36,15 @@ commands =
 #
 description =
     run tests
-    oldestastropy: with astropy 4.0.*
+    oldestdeps: with astropy 4.0.*
     devastropy: with astropy latest
 
 deps =
     devastropy: git+https://github.com/astropy/astropy.git#egg=astropy
-    oldestastropy: astropy==4.0
-
+    oldestdeps: astropy==4.0
+    # We set a suitably old numpy along with an old astropy, no need to pick up
+    # deprecations and errors due to their unmatching versions
+    oldestdeps: numpy==1.16
```

Comment:
How do we match them? This is a question from someone that doesn't use `astropy` very often.

Reply:
check the setup.cfg, or release notes of the oldest astropy you use. So, e.g. when the minimum astropy version is bumped to 5.0, this will need to be bumped too. But this should be noticeable once you have a  contradicting combination.

(This is a bit of a workaround, numpy is not listed as a direct dependency, it's indirectly comes in as astropy's dependency. And not setting it to a minimum version would cause this old version test job to pick up the newest numpy, and that may lead to issues in CI as it may not be compatible (e.g. astropy doesn't have an upper limit for dependencies). Of course users can pick an old astropy with new numpy combination, but I think for testing purposes we should only cover the lower boundaries.)

Reply:
Found it. It's easy with releases being labeled in `astropy`.